main/guides/orchestration/orchestration-basics/ui.md

```diff
@@ -0,0 +1,50 @@
+# DApp User Interface
+
+Here, we will walkthrough the components making up the user interface for the Orchestration dApp.
+
+## Orchestration.tsx
```

Comment:
I would suggest using component name rather than filename for each of these (the component is called `Orchestration`)